bash_completion.sh

```diff
@@ -458,7 +458,7 @@ _buildtest ()
     *)
       local cmds="build buildspec cd cdash clean config debugreport docs help info inspect history path report schema schemadocs stats stylecheck tutorial-examples unittests"
       local alias_cmds="bd bc cg debug it h hy rt style test"
-      local opts="--color --config --debug --editor --help --helpcolor --logpath --loglevel --print-log --no-color --report --version --view-log -c -d -h -l -p -r -V"
+      local opts="--color --config --debug --editor --help --helpcolor --help--all --logpath --loglevel --print-log --no-color --report --version --view-log -c -d -h -l -p -r -V -H"
```

Comment:
Please sort them so put -H before -V thats how it'd setup it helps in tab completion

Reply:
Just did it.